h/static/styles/partials-v2/_search.scss

```diff
@@ -0,0 +1,91 @@
+// Search page
+// -----------
+// Styles for the search page which displays the results
+// of activity queries.
+
+.search {
+  font-size: $normal-font-size;
+  margin-top: 20px;
+  color: $grey-6;
+}
+
+.search-results {
+  list-style: none;
+  width: 950px;
```

Comment:
Don't forget that everything we build for the new site needs to work on mobile. That means that it needs to work on devices with a screen size of 320px and up.

If you want to limit the maximum width, use `max-width` rather than `width` for this reason. Of course, we might need to use a completely different layout for smaller screens, in which case that can be done with media queries at the bottom of the file. See `form-input` for an example.